src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -69,40 +81,24 @@ public ReadsSparkSource(JavaSparkContext ctx) {
      *                  use the default split size (determined by the Hadoop input format, typically the size of one HDFS block).
      * @return RDD of (SAMRecord-backed) GATKReads from the file.
      */
-    public JavaRDD<GATKRead> getParallelReads(final String readFileName, final String referencePath, final List<SimpleInterval> intervals, long splitSize) {
-        final Configuration conf = new Configuration();
+    public JavaRDD<GATKRead> getParallelReads(final String readFileName, final String referencePath, final List<SimpleInterval> intervals, final long splitSize) {
+        final Configuration conf = ctx.hadoopConfiguration();
```

Comment:
I've also been continually confused by the context/confs and what is necessary from them.  If it's necessary that we always use the same one, we should be much more careful about threading it through everywhere that it is needed.  If it's not, can we just always default construct them?  

I suspect that what is important is that we are consistently using the same settings everywhere we need to, and we should commit to either ignoring the local environment by always constructing a new one, or always thread the same one through everywhere.